swagger.yml

```diff
@@ -1225,6 +1225,52 @@ paths:
             - f5
           x-nullable: true
 
+  /metrics/akamai/total-dns-requests:
+    get:
+      tags:
+        - Metrics
+      summary: Get total DNS requests for an Akamai GTM property
+      x-policy: andromeda:metrics:get
+      parameters:
+        - in: query
+          name: domain
+          required: false
+          type: string
+          description: "GTM domain name (default: andromeda.akadns.net)"
+          x-nullable: true
+        - in: query
+          name: property
+          required: true
+          type: string
+          description: GTM property ID
```

Comment:
To be consistent, this should be the domain UUID, not the propory id.
Another reason is - that this API call needs to be verified in the database - e.g. if the domain belongs to the user token according the the goslo policy. The primary key is UUID, which is indexed. You don't want search for domain names.